### PR TITLE
Update Navigation steps and copy - Delete Profile

### PR DESCRIPTION
**Pull request by ThatElfGuy**
Updating steps for Hamburger menu, and correcting the copy since we no longer ask questions about deletion reasons.

### CODE REVIEW

**Comment by Pairadox** [comment]
I find it sad to be removing the fun aspects of the FAQ - it feels like we're losing the FetLife voice and becoming corporate.

File: 02_profiles/02_how-do-i-delete-my-account-profile.md [hunk 1]
```diff
@@ -6,10 +6,11 @@ The easiest way to delete your profile is to go here: [https://fetlife.com/deact
 
 Alternatively, follow these steps to delete or deactivate your profile:
 
-1. Click your nickname in the upper-right corner on any page on FetLife.
-2. Click on "Edit Profile".
-3. On the right side, you'll see a link to "Delete profile".
+1. Click on the hamburger menu (icon with three horizontal lines) on the far right of FetLife's navigation bar.
```
Comment (Pairadox) [inline 02_profiles/02_how-do-i-delete-my-account-profile.md]: upper far right

**Comment by Pairadox** [comment]
One comment added

**Comment by jminion** [comment]
+1

**Pull request merged**